docs/user-guide/admin-cli.md

```diff
@@ -653,17 +701,144 @@ enhancement tracked [here](https://github.com/vmware/docker-volume-vsphere/issue
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy rm --name=some-policy
 Successfully removed policy: some-policy
 ```
+
+## Config
+
+Creates, removes, moves and reports on status of config DB. Config DB keeps authentication information, and without initializing it no access control is supported. Also, configuring access control will fail, e.g.
+```
+[root@localhost:~] vmdkops_admin vm-group create --name MY
+Internal Error(Error: Please init configuration in vmdkops_admin before trying to change it)
+```
+
+#### Init (`config init`)
+
+Initializing the config is optional. If the config is not initialized, there will be no access control and all `vm-group` commands will fail with appropriate messages.
+
+Before configuring access control or quotas, the config needs to be inited to either local (`init --local`) or shared (`init --datastore=ds_name) mode
+
+```
+[root@localhost:~] vmdkops_admin config init -h
+usage: vmdkops_admin.py config init [-h] [--local] [--force]
+                                    [--datastore DATASTORE]
```

Comment:
Agreed, it is not convenient without it (when the data is in `status` only and not `config status`). Added `config status`